include/vl6180_pi.h

```diff
@@ -4,8 +4,29 @@ extern "C"{
 
 typedef int vl6180;
 
-int vl6180_initialise(int device);
+#define VL6180_DEFAULT_ADDR 0x29
+
+///Initialize a vl6180 sensor on the i2c port
+/// \param device The I2C bus to open. e.g. "1" for using /dev/i2c-1
+/// \param i2c_addr addres of the device. If you don't know, pass VL1680_DEFALUT_ADDR to it
+/// \return handle to the sensor. Keep this variable to talk to the sensor via the library
+vl6180 vl6180_initialise(int device, int i2c_addr);
```

Comment:
i would like to keep this backwards compatible if possible. Can you add another initialise function that defaults to the defined address

Reply:
Fair enough.

What I'm gonna do is rename this function "vl6480_initialise_addr", and keep the "vl6480_initialise" with the old interface, and make it call this one with default address that is now defined in the header.

I will update the PR shortly ;-)